### PR TITLE
IPFS-scholar

### PR DESCRIPTION
**Pull request by liyue634**
Application submitted by IPFs-Scholar.

### CODE REVIEW

File: IPFS-scholar.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# <Project Name>
+
+Before submitting a PR containing your team's project information:
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+## (0) Project Name and Team Slack Handles
+
+*IPFS-scholar/hfs-ipfs-scholar*
```
Comment (pooja) [inline IPFS-scholar.md:9]: Hey there! Would you be able to include your individual Filecoin slack handle here as well so we can contact you on Slack if needed?

**Comment by dineshshenoy** [comment]
Before reviewing the application, can you resubmit as a single template addition?  This PR seems to add a number of extraneous files and removes some of the administrative files for the competition.   Thanks!

File: IPFS-scholar.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+*Scholars can publish their research results through IPFs-Scholar, and they can also download some papers and data sets from websites and reward the authors.*
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+**
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+
+*All the data we collect is open source and free, and all uploaded data are public data.*
```
Comment (dineshshenoy) [inline IPFS-scholar.md:29]: Can you specify the source of this data?

Comment (liyue634) [reply IPFS-scholar.md:29]: We have a team that collects free and open datasets and other academic materials in astromedicine. And we have reached a preliminary agreement with a medical institution in the UK to store hospital data on our platform.

File: IPFS-scholar.md (hunk 1 above)
Comment (liyue634) [reply IPFS-scholar.md:9]: I have updated the information in the new submission. Please check.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
+{
```
Comment (GregTheGreek) [inline package.json:1]: It looks like you committed your entire project. Please remove everything except for the application file (`.md`)

**Review by GregTheGreek — CHANGES_REQUESTED** [review]

**Comment by dineshshenoy** [comment]
Can you simplify the submission to a single additive file with your submission?   This PR removes some key data from the repo.

**Comment by liyue634** [comment]
I have resubmitted it, please review it, thank you!




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "dineshshenoy"<notifications@github.com&gt;; 
发送时间: 2020年9月26日(星期六) 凌晨5:57
收件人: "filecoin-project/slingshot"<slingshot@noreply.github.com&gt;; 
抄送: "青年"<634628730@qq.com&gt;; "Assign"<assign@noreply.github.com&gt;; 
主题: Re: [filecoin-project/slingshot] IPFS-scholar (#17)





 
您可以将提交的提交简化为一个附加文件吗？此PR从存储库中删除了一些关键数据。
 
— 
您收到此邮件是因为您已被分配。
直接回复此电子邮件，在GitHub上查看，或取消订阅。

**Comment by liyue634** [comment]
I have resubmitted it, please review it, thank you!




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Gregory Markou"<notifications@github.com&gt;; 
发送时间: 2020年9月25日(星期五) 晚上11:25
收件人: "filecoin-project/slingshot"<slingshot@noreply.github.com&gt;; 
抄送: "青年"<634628730@qq.com&gt;; "Assign"<assign@noreply.github.com&gt;; 
主题: Re: [filecoin-project/slingshot] IPFS-scholar (#17)





 
@GregTheGreek请求对此请求请求进行更改。
 
 
在package.json中：
 &gt; @@ -0,0 +1,27 @@ +{  
看起来您已经提交了整个项目。请删除除应用程序文件（.md）以外的所有内容
 
— 
您收到此邮件是因为您已被分配。
直接回复此电子邮件，在GitHub上查看，或取消订阅。

**Comment by cwaring** [comment]
duplicate of #71 & #90

**Pull request closed without merging**